docker-release.sh

```diff
@@ -54,12 +66,13 @@ curl -u $user:$pass -i -X DELETE \
 # Deploy images for various DB backends
 for dbtag in "${dbtags[@]}"
 do
+  name="$(containerName $dbtag)"
```

Comment:
likewise, it's going to fail because of space and because `tinode alldbs` won't exist.